chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -139,8 +143,13 @@ async def instantiate_mempool_manager(
     block_timestamp: uint64 = TEST_TIMESTAMP,
     constants: ConsensusConstants = DEFAULT_CONSTANTS,
     max_tx_clvm_cost: Optional[uint64] = None,
+    get_unspent_lineage_info_for_puzzle_hash: Callable[
+        [bytes32], Awaitable[Optional[UnspentLineageInfo]]
+    ] = zero_calls_get_unspent_lineage_info_for_puzzle_hash,
 ) -> MempoolManager:
-    mempool_manager = MempoolManager(get_coin_records, constants, max_tx_clvm_cost=max_tx_clvm_cost)
+    mempool_manager = MempoolManager(
+        get_coin_records, get_unspent_lineage_info_for_puzzle_hash, constants, max_tx_clvm_cost=max_tx_clvm_cost
```

Comment:
why not just pass `zero_calls_get_unspent_lineage_info_for_puzzle_hash` here directly?

Reply:
Hmm I didn't want to alter all `instantiate_mempool_manager` calls and increase the patch size. My rationale being if any caller needs an explicit function here, it can pass one.

Reply:
I don't see you alter *any* call to `instantiate_mempool_manager()`. are you?
as far as I can tell, all calls to `instantiate_mempool_manager()` will currently use the `zero_calls_get_unspent_lineage_info_for_puzzle_hash`. Is that not the case?

If you would remove the parameter you added, and pass in `zero_calls_get_unspent_lineage_info_for_puzzle_hash` unconditionally, that would have the same effect, wouldn't it?